src/verilog.y

```diff
@@ -5481,10 +5485,10 @@ elseStmtBlock<nodep>:   // Part of concurrent_assertion_statement
 //UNSUP         property_port_listE                     { $$ = $1; }
 //UNSUP ;
 
-//UNSUPproperty_formal_typeNoDt<nodep>:  // IEEE: property_formal_type (w/o implicit)
+//property_formal_typeNoDt<nodep>:  // IEEE: property_formal_type (w/o implicit)
 //UNSUP         sequence_formal_typeNoDt                { $$ = $1; }
-//UNSUP |       yPROPERTY                               { }
-//UNSUP ;
+        // |       yPROPERTY                               { }
+        // ;
```

Comment:
Is this now unused?  Confused why is empty rule.

Reply:
It looks that I removed //UNSUP comments at the beginning of work on it and then just forgot to restore them.